.prettierrc

```diff
@@ -1,10 +1,10 @@
 {
 	"printWidth": 80,
-	"tabWidth": 2,
-	"useTabs": true,
 	"semi": false,
 	"singleQuote": true,
+	"tabWidth": 2,
 	"trailingComma": "es5",
+	"useTabs": true,
```

Comment:
Hi, 
I was surprised to see tabs in the project. The standard is 2 spaces. Don't you want to benefit of this PR to set 2 spaces?